File: tests/fixtures/python/sample.py
MAX_RETRIES = 3

class BaseService:
    """Base service class."""
    def health_check(self) -> bool:
        """Check service health."""
        return True

class UserService(BaseService):
    """Manages user operations."""
    def get_user(self, user_id: int) -> dict:
        """Get user by ID."""
        return {"id": user_id}

    def delete_user(self, user_id: int) -> bool:
        """Delete a user."""
        return True

def authenticate(token: str) -> bool:
    """Authenticate a token."""
    return len(token) > 0

File: tests/fixtures/java/Sample.java
package sample;

import java.io.Serializable;

/**
 * Base service
 */
public abstract class BaseService {
    public abstract void init();
}

/**
 * Repository contract
 */
public interface Repository {
    String findById(int id);
}

/**
 * User service
 */
public class Sample extends BaseService implements Serializable, Repository {
    public static final int MAX_RETRIES = 3;

    public void init() {}

    public String findById(int id) {
        return "user-" + id;
    }

    /**
     * Get user by ID
     */
    public String getUser(int userId) {
        return "user-" + userId;
    }

    public static boolean authenticate(String token) {
        return token != null && !token.isEmpty();
    }
}
